Firestore/Example/Tests/Local/FSTRemoteDocumentCacheTests.m

```diff
@@ -114,6 +114,7 @@ - (void)testDocumentsMatchingQuery {
   FSTDocumentDictionary *results = [self.remoteDocumentCache documentsMatchingQuery:query];
   NSArray *expected =
       @[ FSTTestDoc(@"b/1", kVersion, _kDocData, NO), FSTTestDoc(@"b/2", kVersion, _kDocData, NO) ];
+  XCTAssertEqual([results count], [expected count]);
   for (FSTDocument *doc in expected) {
     XCTAssertEqualObjects([results objectForKey:doc.key], doc);
   }
```

Comment:
The TODO below is now no longer relevant, is it?

Reply:
Nope. Removed.